docs/website/mtouch-errors.md

```diff
@@ -1531,6 +1531,55 @@ In some cases, it's a "local network" issue and it can be addressed by adding th
 
 <h3><a name="MT7044"/>MT7044: Main Info.plist path does not specify a UIDeviceFamily.</h3>
 
+<h3><a name="MT7045"/>MT7045: Unrecognized Type: *.</h3>
+
```

Comment:
also the error message should stay as close as possible to what's reported (to avoid confusion).

Title: `Unrecognized * Format`
Description: where `*` can be `Real` (float), `Integer` ...